src/components/header/header.tsx

```diff
@@ -0,0 +1,36 @@
+import Logo from '../logo/logo.tsx';
+
+function Header(): JSX.Element{
+  return (
+    <header className="header">
+      <div className="container">
+        <div className="header__wrapper">
+          <div className="header__left">
+            <Logo blockName={'header'} active/>
```

Comment:
Для строк не нужна интерполяция
```suggestion
            <Logo blockName="header" active/>
```